
r_il: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
t_il: [0.04165, 0.02326, -0.0284]
r_cl: [0.00162756, -0.999991, 0.00390957, -0.0126748, -0.00392989, -0.999912, 0.999918, 0.00157786, -0.012681]
t_cl: [0.0409257, 0.0318424, -0.0927219]
